src/components/notesCard/notescard.css

```diff
@@ -0,0 +1,32 @@
+.notes-card{
+    display: flex;
+    flex-direction: column;
+    max-width: 15rem;
+    max-height: fit-content;
+    padding: 0.5rem;
+    padding-top: 1rem;
+    border-radius: 10px;
+    border: 0.1rem solid var(--primary);
+    color: var(--font-primary);
+    font-weight: 500;
+}
+.note-title{
+
+    margin-bottom: 0.5rem;
+}
+.note-action-icons{
+    margin-top: 1rem;
```

Comment:
suggestion: have an empty line after each block of code, makes code more readable